src/components/grid/ProjectGrid.vue

```diff
@@ -1,47 +1,68 @@
 <template>
   <div class="project-grid">
-    <div>
-      <div class="row items-center">
-        <h4>
-          {{ $t('page.home.project.title') }}
-        </h4>
+    <div class="row items-center">
+      <h4 class="q-ma-md q-mr-xl">
+        {{ $t('page.home.project.title') }}
+      </h4>
+      <q-btn
+        outline
+        no-caps
+        class="q-ml-sm"
+        color="primary"
+        icon="fa-solid fa-cloud-arrow-down"
+        :label="$t('actions.home.importProject')"
+        data-cy="import-project-button"
+        @click="DialogEvent.next({ type: 'open', key: 'ImportProject' })"
+      />
+    </div>
+    <div class="row">
+      <h5 class="text-secondary q-ma-md">
+        {{ $t('page.home.project.recent') }}
+      </h5>
+    </div>
+    <div class="row no-wrap">
+      <div class="row">
         <q-btn
           outline
           no-caps
-          class="q-ml-xl"
           color="primary"
-          icon="fa-solid fa-plus"
-          :label="$t('actions.home.newProject')"
+          class="square-button"
```

Comment:
could be more explicit like "create-project-button" or "new-project-button"

Reply:
done :heart: